com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Multiply blend now keeps DstAlpha as it's RGB only.
 - Particle AlphaModulate() renamed to AlphaModulateAndPremultiply() as it does both. Moved separate AlphaModulate() and AlphaPremultiply() to URP shader library. Fix double alpha multiply for ParticleLit.
 - Improved blending modes trigger a material update which tries to keep the existing look intact. This is not always possible and manual blend mode changes might be required.
+- Reverted intermediate texture behavior.
```

Comment:
This might be confusing to users, as it's also not very descriptive as well. 
 Maybe it should also be a good idea to revert the original changelog the intermediate RT texture added?

Reply:
I think we should leave those be, as those versions will have the behavior. But yeah, it does sound confusing. I'll make it sound a bit nicer :)